src/test/java/qna/domain/AnswerTest.java

```diff
@@ -1,6 +1,56 @@
 package qna.domain;
 
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import qna.repository.AnswerRepository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static qna.domain.UserTest.JAVAJIGI;
+import static qna.domain.UserTest.SANJIGI;
+
+@DataJpaTest
 public class AnswerTest {
-    public static final Answer A1 = new Answer(UserTest.JAVAJIGI, QuestionTest.Q1, "Answers Contents1");
-    public static final Answer A2 = new Answer(UserTest.SANJIGI, QuestionTest.Q1, "Answers Contents2");
+
+    public static final Answer A1 = new Answer(JAVAJIGI, QuestionTest.Q1, "Answers Contents1");
+    public static final Answer A2 = new Answer(SANJIGI, QuestionTest.Q1, "Answers Contents2");
+
+    @Autowired
+    private AnswerRepository answerRepository;
+
+    @Test
+    @DisplayName("Answer 객체를 저장하면 Id가 자동생성 되어 Not Null 이다.")
+    void save() {
+        assertThat(A1.getId()).isNotNull();
+        assertThat(answerRepository.save(A1).getId()).isNotNull();
+        assertThat(answerRepository.save(A2).getId()).isNotNull();
+    }
+
+    @Test
+    @DisplayName("Answer 객체를 조회하면 데이터 여부에 따라 Optional 존재 여부가 다르다." +
+            "또한 동일한 객체면 담긴 값도 동일하다.")
+    void findByWriterId() {
+        answerRepository.save(A1);
+        assertThat(answerRepository.findByWriterId(1L).isPresent()).isTrue();
+        assertThat(answerRepository.findByWriterId(1L).get().getContents()).isEqualTo(A1.getContents());
+        assertThat(answerRepository.findByWriterId(10L).isPresent()).isFalse();
```

Comment:
assertj 를 활용하시면 `Optional` 에 대한 검증과 메서드 체이닝을 통해 검증이 가능한데요.
다음과 같이 검증도 가능하니 참고만 해주세요 🙏

```suggestion
        assertThat(questionRepository.findByWriterId(1L)).isPresent()
                .get()
                .extracting(Question::getContents)
                .isEqualTo(Q1.getContents());
        assertThat(questionRepository.findByWriterId(10L)).isEmpty();
```